docs/usage.md

```diff
@@ -202,7 +208,8 @@ process {
 }
 ```
 
-> **NB:** We specify just the process name i.e. `STAR_ALIGN` in the config file and not the full task name string that is printed to screen in the error message or on the terminal whilst the pipeline is running i.e. `RNASEQ:ALIGN_STAR:STAR_ALIGN`. You may get a warning suggesting that the process selector isn't recognised but you can ignore that if the process name has been specified correctly. This is something that needs to be fixed upstream in core Nextflow.
+> **NB:** We specify just the process name i.e. `STAR_ALIGN` in the config file and not the full task name string that is printed to screen in the error message or on the terminal whilst the pipeline is running i.e. `RNASEQ:ALIGN_STAR:STAR_ALIGN`.
+> You may get a warning suggesting that the process selector isn't recognised but you can ignore that if the process name has been specified correctly. This is something that needs to be fixed upstream in core Nextflow.
```

Comment:
This should be a real warning now given the changes to Nextflow. It's unlikely one would try and change configuration of a process not executed (i.e. behind an `if`), so if the `WARN` shows up, then it is likely a real spelling mistake or something.

Reply:
👍🏽 Both of these changes have come in as a result of the template update so we should change things there. Fancy doing that there and I will pull in the changes here? Off today but can take a look tomorrow.

In fact, I would prefer if we move all of this stuff out of the pipeline and onto the website if possible and just link there. Will make it easier to update in one place. Something we should definitely tackle during the Hackathon.

Reply:
Made a pull to the template for now. https://github.com/nf-core/tools/pull/1412

Reply:
Thanks! Pulled in those changes in https://github.com/nf-core/rnaseq/pull/767/commits/b7a34cb2392aa171bbf7b13177e1fe71aae29bc1 https://github.com/nf-core/rnaseq/pull/767/commits/d04519cce8cbac89b76909247a6f8f2b115dfefe